src/diff.jl

```diff
@@ -629,6 +635,24 @@ let
     end
 end
 
+"""
+```julia
+hessian_sparsity(f,input::AbstractVector{T}, args...;kwargs...) where T<:Number
+```
+
+Return the sparsity pattern of the Hessian of the function `f(input,args...;kwargs...)`.
+"""
```

Comment:
The docstring for this new method is not shown in the documentation. You need to move it to the existing docstring of `function hessian_sparsity end`.

Reply:
It would be good in the docpage to have a subsection "high level dispatches" which lists these two, and then "symbolic dispatches" which lists the old ones. I presume the "I'll give you a function, give me the sparsity" high level dispatches are the most useful, so it would be good to highlight them like that.